drivers/lte_link_control/lte_lc.c

```diff
@@ -126,6 +149,7 @@ static int w_lte_lc_init_and_connect(struct device *unused)
 				break;
 			}
 		}
+		k_sleep(K_MSEC(10));
```

Comment:
Why is this needed? Is blocking `recv` misbehaving?

Reply:
yes, this is related to missing implementation of bsd_os_timedwait.

CC @jtguggedal

Reply:
Ok, makes sense, thanks for claryfing